__mocks__/next/image.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react'
 import type { ImageProps } from 'next/image'
+import _ from 'lodash'
 
-const Image = (props: ImageProps) => <img {...props} />
+const Image = (props: ImageProps) => {
+  //not all Image properties are allowed in <img/> tag
+  const copy = _.omit(props, 'objectFit')
```

Comment:
Why not simply delete that property? Making a copy and using lodash seems kinda overkill.

```suggestion
  delete props.objectFit
```

Or if the copy is really needed you can do it with vanilla JS.

```suggestion
  const { objectFit, ...copy } = props
```

Reply:
`delete` doesn't work, maybe it's inherited from the parent or protected/frozen. Simple spread is probably more readable I agree.